src/liboslexec/llvm_util.cpp

```diff
@@ -2448,17 +2448,26 @@ LLVM_Util::make_function(const std::string& name, bool fastcall,
                          llvm::Type* rettype, cspan<llvm::Type*> params,
                          bool varargs)
 {
+    ustring function_prefix;
+    if (m_function_prefixes.size())
```

Comment:
Maybe add comment here as to "why" we are doing this whole prefix thing.  Fresh reader might be confused.